arduino/RobotinoCommunication/RobotinoCommunication.ino

```diff
@@ -0,0 +1,34 @@
+#include <avr/interrupt.h>
+
+void setup() {
+  pinMode(2, INPUT);
+  pinMode(3, INPUT);
+  pinMode(4, INPUT);
+  pinMode(5, INPUT);
+  pinMode(6, INPUT);
+  pinMode(7, INPUT_PULLUP);
+  Serial.begin(9600);
+
+  cli();
+  PCICR |= 0b00000100;
+  PCMSK2 |= 0b10000000;
+  sei();
+}
+
+void loop() {
+}
+
+ISR(PCINT2_vect) {
+  if(digitalRead(7) == HIGH){
+    byte value = 0;
```

Comment:
value global definieren, damit diese in der Loop ausgewertet werden kann.

Reply:
den Zustand in einer Merkervariablen speichern, damit in der Loop die Messung nur durchgeführt wird, wenn der Merker = true ist.